src/main/resources/static/scripts/login.js

```diff
@@ -1,3 +1,31 @@
 function login() {
-    window.location.href = 'main.html';
+
+    var emailElement=document.getElementById("emailInput");
```

Comment:
It's better to use lat instead of var
https://www.devkozak.com/pytania-js/#5_Czym_sie_rozni_deklaracja_zmiennych_przez_var_let_oraz_const_Jakie_maja_zakresy
same in register.js

Reply:
I wonder how is that possible that variable declared using let is available for `fetch`. In my opinion that's different block.